team_unified_prs.rb

```diff
@@ -42,7 +42,7 @@ def positional_arg_or_env_split(idx, env_var)
 
   add_prs = proc do |prs|
     for pr in prs
-      next if skip_pr_ids.include?(pr["url"])
+      next if skip_pr_ids.include?(pr["url"]) || pr["archived"]
```

Comment:
This reads like the PR is archived, `repoArchived` or `repoIsArchived` or similar would probably be a better key

Reply:
updated